lib/rails_event_store/client.rb

```diff
@@ -1,7 +1,7 @@
 module RailsEventStore
   class Client
 
-    def initialize(repository = Repositories::EventRepository.new)
+    def initialize(repository = ::RailsEventStore::Repository.backend)
```

Comment:
Why not just be explicit and let developer to specify repository here? 
Then we won't need global backend configuration and meta code in `lib/rails_event_store/repository.rb`.